R/tokenization.R

```diff
@@ -656,3 +656,73 @@ apply_to_chars <- function(text, .f, ...) {
     collapse = ""
   )
 }
+
+
+# tokenize_text -----------------------------------------------------------
+
+#' Tokenize Text with Word Pieces
+#'
+#' Given some text and a word piece vocabulary, tokenizes the text. This is
+#' primarily a tool for quickly checking the tokenization of a piece of text.
+#'
+#' @param text Character vector; text to tokenize.
+#' @param ckpt_dir Character; path to checkpoint directory. If specified,
```

Comment:
Same inheritParams suggestions as above.

Reply:
Yup, this is the place I *did* have it, then took it out.